### PR TITLE
Feat/54 new clustering option

### PR DESCRIPTION
**Pull request by f-lukas**
implementation of https://github.com/Pandora-IsoMemo/iso-app/issues/54
fix of https://github.com/Pandora-IsoMemo/iso-app/issues/108

### CODE REVIEW

File: R/03-modelResults2DKernel.R [hunk 1]
```diff
@@ -63,11 +63,15 @@ modelResults2DKernelUI <- function(id, title = "", asFruitsTab = FALSE){
           selectInput(inputId = ns("Weighting"),
                       label = "Weighting variable (optional):",
                       choices = c("")),
-          checkboxInput(inputId = ns("kMeans"),
-                        label = "Do k-means clustering",
-                        value = FALSE, width = "100%"),
```
Comment (arunge) [inline R/03-modelResults2DKernel.R:68]: Before there was the option to run the model without clustering:

image

Are now clusters always calculated? Would it make sense to add an option "none" and use this per default? (If so, the slider "No. of clusters" should be hidden.)

image

File: NEWS.md [hunk 2]
```diff
@@ -1,5 +1,12 @@
 # MpiIsoApp development version
 
+## Version 23.03.4
+
+### Updates
```
Comment (arunge) [inline NEWS.md:5]:
```suggestion
### New Features
```

File: NEWS.md [hunk 3]
```diff
@@ -1,5 +1,12 @@
...
+- Adds mclust as new clustering option for KernelR and KernelTimeR
+- Unifies colors for clustering map and clustering plot within KernelTimeR
```
Comment (arunge) [inline NEWS.md:7]:
```suggestion

```

**Review by arunge — CHANGES_REQUESTED** [review]
Looks fine and seems to work. :+1: Just some minor remarks and questions.

File: R/03-modelResults2DKernel.R (hunk 1 above)
Comment (f-lukas) [reply R/03-modelResults2DKernel.R:68]: You can still run it without clustering if you remove the selection in Cluster Method (using backspace). This is the default when you start the app. Do you think another option "none" is more intuitive. The way it is currently implemented is the same as for the previous inputs (e.g. Weighting variable (optional):)

Comment (arunge) [reply R/03-modelResults2DKernel.R:68]: Ok, then keep it as it is to be consistent with the other input fields. :+1:

**Comment by arunge** [comment]
merged withi
- https://github.com/Pandora-IsoMemo/iso-app/pull/129

**Pull request closed without merging**